source/sysadmins-guide/varnish-setup/index.md

```diff
@@ -150,6 +150,13 @@ sub vcl_recv {
     # Mitigate httpoxy application vulnerability, see: https://httpoxy.org/
     unset req.http.Proxy;
 
+    # Strip query strings only needed by browser javascript
+    if (req.url ~ "(\?|&)(gclid|utm_[a-z]+)=") {
+        # see rfc3986#section-2.3 "Unreserved Characters" for regex
+        set req.url = regsuball(req.url, "(gclid|utm_[a-z]+)=[A-Za-z0-9\-\_\.\~]+&?", "");
```

Comment:
Could you consider all ignoring default attributes? https://github.com/shopware/shopware/blob/5.4/engine/Shopware/Configs/Default.php#L163

Reply:
best case, implementors evaluate the config file and customize to their needs